pkg/apis/storage/types.go

```diff
@@ -390,6 +390,24 @@ type CSIDriverSpec struct {
 	//
 	// +optional
 	RequiresRepublish *bool
+
+	// SELinuxMount specifies if the CSI driver supports "-o context"
+	// mount option.
+	//
+	// When "true", the CSI driver must ensure that all volumes provided by this CSI
+	// driver can be mounted separately with different `-o context` options. This is
+	// typical for storage backends that provide volumes as filesystems on block
+	// devices or as independent shared volumes.
+	// Kubernetes *may* call NodeStage / NodePublish with "-o context=xyz" mount
+	// option when it considers the volume is used only with a single SELinux context.
```

Comment:
is this essentially RWOP access mode, or is it more complicated than that?

Reply:
Right now it's for RWOP, but in the future we plan to open it to all pods that use the same volume with the same SELinux context on the same node. I am not sure if API description should be that detailed.

Reply:
I think it's probably worth mentioning and then update it if it changes in the future. I think otherwise people could get confused when they don't see the behavior happening and can't tell if it's a bug or intended behavior.

Reply:
Added some note about RWOP, PTAL